src/assets/locales/de/main.json

```diff
@@ -137,8 +137,10 @@
         "BUSINESS_PARTNER_NUMBER": "BPN Creation"
       },
       "status": {
-        "TO_DO": "To Do",
-        "FAILED": "Failed"
+        "IN_PROGRESS": "Im Gange",
```

Comment:
Please rename IN_PROGRESS to "In Bearbeitung"

Reply:
@jjeroch Made the changes